2021-08-11 23:28:25,729 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:28:29,942 INFO server.py Установлено соедение с ПК ('127.0.0.1', 52028)
2021-08-11 23:28:29,942 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1628713709.9423618, 'user': {'account_name': 'Guest'}}
2021-08-11 23:28:29,942 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1628713709.9423618, 'user': {'account_name': 'Guest'}}
2021-08-11 23:28:29,942 INFO server.py Cформирован ответ клиенту {'response': 200}
2021-08-11 23:28:29,943 DEBUG server.py Соединение с клиентом ('127.0.0.1', 52028) закрывается.
2021-08-11 23:29:09,056 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:29:10,852 INFO server.py Установлено соедение с ПК ('127.0.0.1', 52047)
2021-08-11 23:29:10,852 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1628713750.8521519, 'user': {'account_name': 'Guest'}}
2021-08-11 23:29:10,852 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1628713750.8521519, 'user': {'account_name': 'Guest'}}
2021-08-11 23:29:10,852 DEBUG server.py Сообщение action от user успешен
2021-08-11 23:29:10,853 INFO server.py Cформирован ответ клиенту {'response': 200}
2021-08-11 23:29:10,853 DEBUG server.py Соединение с клиентом ('127.0.0.1', 52047) закрывается.
2021-08-11 23:35:45,089 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:35:46,996 INFO server.py Установлено соедение с ПК ('127.0.0.1', 52231)
2021-08-11 23:35:46,996 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1628714146.9962492, 'user': {'account_name': 'Guest'}}
2021-08-11 23:35:46,996 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714146.9962492, 'user': {'account_name': 'Guest'}}
2021-08-11 23:35:46,997 DEBUG server.py Формирование presence от account_name успешено
2021-08-11 23:35:46,997 INFO server.py Cформирован ответ клиенту {'response': 200}
2021-08-11 23:35:46,997 DEBUG server.py Соединение с клиентом ('127.0.0.1', 52231) закрывается.
2021-08-11 23:37:52,686 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:37:57,079 INFO server.py Установлено соедение с ПК ('127.0.0.1', 52301)
2021-08-11 23:37:57,079 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1628714277.0790858, 'user': {'account_name': 'Guest'}}
2021-08-11 23:37:57,079 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714277.0790858, 'user': {'account_name': 'Guest'}}
2021-08-11 23:37:57,079 DEBUG server.py Формирование presence от {'account_name': 'Guest'} успешно
2021-08-11 23:37:57,079 INFO server.py Cформирован ответ клиенту {'response': 200}
2021-08-11 23:37:57,079 DEBUG server.py Соединение с клиентом ('127.0.0.1', 52301) закрывается.
2021-08-11 23:38:37,188 INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:38:38,744 INFO server.py Установлено соедение с ПК ('127.0.0.1', 52320)
2021-08-11 23:38:38,744 DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1628714318.7449205, 'user': {'account_name': 'Guest'}}
2021-08-11 23:38:38,744 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714318.7449205, 'user': {'account_name': 'Guest'}}
2021-08-11 23:38:38,744 DEBUG server.py Формирование presence от Guest успешно
2021-08-11 23:38:38,744 INFO server.py Cформирован ответ клиенту {'response': 200}
2021-08-11 23:38:38,744 DEBUG server.py Соединение с клиентом ('127.0.0.1', 52320) закрывается.
2021-08-11 23:39:59,042       INFO server.py Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:40:00,506       INFO server.py Установлено соедение с ПК ('127.0.0.1', 52374)
2021-08-11 23:40:00,506      DEBUG server.py Получено сообщение {'action': 'presence', 'time': 1628714400.5057054, 'user': {'account_name': 'Guest'}}
2021-08-11 23:40:00,506      DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714400.5057054, 'user': {'account_name': 'Guest'}}
2021-08-11 23:40:00,506      DEBUG server.py Формирование presence от Guest успешно
2021-08-11 23:40:00,506       INFO server.py Cформирован ответ клиенту {'response': 200}
2021-08-11 23:40:00,506      DEBUG server.py Соединение с клиентом ('127.0.0.1', 52374) закрывается.
2021-08-11 23:41:03,131       INFO server.py----- Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:41:04,318       INFO server.py----- Установлено соедение с ПК ('127.0.0.1', 52401)
2021-08-11 23:41:04,318      DEBUG server.py----- Получено сообщение {'action': 'presence', 'time': 1628714464.3176486, 'user': {'account_name': 'Guest'}}
2021-08-11 23:41:04,318      DEBUG server.py----- Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714464.3176486, 'user': {'account_name': 'Guest'}}
2021-08-11 23:41:04,318      DEBUG server.py----- Формирование presence от Guest успешно
2021-08-11 23:41:04,318       INFO server.py----- Cформирован ответ клиенту {'response': 200}
2021-08-11 23:41:04,318      DEBUG server.py----- Соединение с клиентом ('127.0.0.1', 52401) закрывается.
2021-08-11 23:42:27,087 INFO server.py ----- Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:42:28,382 INFO server.py ----- Установлено соедение с ПК ('127.0.0.1', 52442)
2021-08-11 23:42:28,382 DEBUG server.py ----- Получено сообщение {'action': 'presence', 'time': 1628714548.3817542, 'user': {'account_name': 'Guest'}}
2021-08-11 23:42:28,382 DEBUG server.py ----- Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714548.3817542, 'user': {'account_name': 'Guest'}}
2021-08-11 23:42:28,382 DEBUG server.py ----- Формирование presence от Guest успешно
2021-08-11 23:42:28,382 INFO server.py ----- Cформирован ответ клиенту {'response': 200}
2021-08-11 23:42:28,382 DEBUG server.py ----- Соединение с клиентом ('127.0.0.1', 52442) закрывается.
2021-08-11 23:42:55,170 ----- INFO server.py ----- Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:42:56,636 ----- INFO server.py ----- Установлено соедение с ПК ('127.0.0.1', 52454)
2021-08-11 23:42:56,636 ----- DEBUG server.py ----- Получено сообщение {'action': 'presence', 'time': 1628714576.6350923, 'user': {'account_name': 'Guest'}}
2021-08-11 23:42:56,636 ----- DEBUG server.py ----- Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714576.6350923, 'user': {'account_name': 'Guest'}}
2021-08-11 23:42:56,636 ----- DEBUG server.py ----- Формирование presence от Guest успешно
2021-08-11 23:42:56,636 ----- INFO server.py ----- Cформирован ответ клиенту {'response': 200}
2021-08-11 23:42:56,636 ----- DEBUG server.py ----- Соединение с клиентом ('127.0.0.1', 52454) закрывается.
2021-08-11 23:43:27,382 ----- INFO server.py ----- Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:43:28,726 ----- INFO server.py ----- Установлено соедение с ПК ('127.0.0.1', 52468)
2021-08-11 23:43:28,727 ----- DEBUG server.py ----- Получено сообщение {'action': 'presence', 'time': 1628714608.7260447, 'user': {'account_name': 'Guest'}}
2021-08-11 23:43:28,727 ----- DEBUG server.py ----- Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714608.7260447, 'user': {'account_name': 'Guest'}}
2021-08-11 23:43:28,727 ----- DEBUG server.py ----- Формирование presence от Guest успешно
2021-08-11 23:43:28,727 ----- INFO server.py ----- Cформирован ответ клиенту {'response': 200}
2021-08-11 23:43:28,727 ----- DEBUG server.py ----- Соединение с клиентом ('127.0.0.1', 52468) закрывается.
2021-08-11 23:43:56,868 ----- INFO       server.py ----- Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-11 23:43:56,998 ----- INFO       server.py ----- Установлено соедение с ПК ('127.0.0.1', 52480)
2021-08-11 23:43:56,999 ----- DEBUG      server.py ----- Получено сообщение {'action': 'presence', 'time': 1628714636.998825, 'user': {'account_name': 'Guest'}}
2021-08-11 23:43:56,999 ----- DEBUG      server.py ----- Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714636.998825, 'user': {'account_name': 'Guest'}}
2021-08-11 23:43:56,999 ----- DEBUG      server.py ----- Формирование presence от Guest успешно
2021-08-11 23:43:57,000 ----- INFO       server.py ----- Cформирован ответ клиенту {'response': 200}
2021-08-11 23:43:57,000 ----- DEBUG      server.py ----- Соединение с клиентом ('127.0.0.1', 52480) закрывается.
2021-08-11 23:43:59,625 ----- INFO       server.py ----- Установлено соедение с ПК ('127.0.0.1', 52483)
2021-08-11 23:43:59,625 ----- DEBUG      server.py ----- Получено сообщение {'action': 'presence', 'time': 1628714639.6251836, 'user': {'account_name': 'Guest'}}
2021-08-11 23:43:59,625 ----- DEBUG      server.py ----- Разбор сообщения от клиента : {'action': 'presence', 'time': 1628714639.6251836, 'user': {'account_name': 'Guest'}}
2021-08-11 23:43:59,625 ----- DEBUG      server.py ----- Формирование presence от Guest успешно
2021-08-11 23:43:59,625 ----- INFO       server.py ----- Cформирован ответ клиенту {'response': 200}
2021-08-11 23:43:59,625 ----- DEBUG      server.py ----- Соединение с клиентом ('127.0.0.1', 52483) закрывается.
2021-08-11 23:47:57,839 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-11 23:47:57,839 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-11 23:47:57,839 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-11 23:47:57,839 ----- INFO       config_server_log.py ----- Информационное сообщение
2021-08-11 23:48:27,470 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-11 23:48:27,470 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-11 23:48:27,470 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-11 23:48:27,470 ----- INFO       config_server_log.py ----- Информационное сообщение
2021-08-11 23:48:58,687 ----- CRITICAL   config_server_log.py ----- Критическая ошибка

2021-08-11 23:48:58,687 ----- ERROR      config_server_log.py ----- Ошибка

2021-08-11 23:48:58,687 ----- DEBUG      config_server_log.py ----- Отладочная информация

2021-08-11 23:48:58,687 ----- INFO       config_server_log.py ----- Информационное сообщение

2021-08-12 00:18:23,442 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-12 00:18:23,442 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-12 00:18:23,442 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-12 00:18:23,442 ----- INFO       config_server_log.py ----- Информационное сообщение
2021-08-12 00:18:23,442 ----- DEBUG      config_server_log.py ----- Тестовый запуск
2021-08-12 00:20:07,152 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-12 00:20:07,153 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-12 00:20:07,153 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-12 00:20:07,153 ----- INFO       config_server_log.py ----- Информационное сообщение
2021-08-12 00:20:07,153 ----- DEBUG      config_server_log.py ----- Тестовый запуск
2021-08-12 00:21:43,371 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-12 00:21:43,371 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-12 00:21:43,371 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-12 00:21:43,371 ----- INFO       config_server_log.py ----- Информационное сообщение
2021-08-12 00:21:43,372 ----- DEBUG      config_server_log.py ----- Тестовый запуск
2021-08-12 00:23:19,567 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-12 00:23:19,567 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-12 00:23:19,567 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-12 00:23:19,567 ----- INFO       config_server_log.py ----- Информационное сообщение
2021-08-12 00:23:28,896 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-12 00:23:28,896 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-12 00:23:28,897 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-12 00:23:28,897 ----- INFO       config_server_log.py ----- Информационное сообщение
2021-08-12 00:23:31,005 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-12 00:23:31,005 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-12 00:23:31,005 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-12 00:23:31,005 ----- INFO       config_server_log.py ----- Информационное сообщение
2021-08-12 00:24:26,672 ----- INFO       server.py ----- Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-12 00:24:28,320 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-12 00:24:28,320 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-12 00:24:28,320 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-12 00:24:28,320 ----- INFO       config_server_log.py ----- Информационное сообщение
2021-08-12 00:25:38,902 ----- INFO       server.py ----- Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2021-08-12 00:25:40,334 ----- CRITICAL   config_server_log.py ----- Критическая ошибка
2021-08-12 00:25:40,334 ----- ERROR      config_server_log.py ----- Ошибка
2021-08-12 00:25:40,334 ----- DEBUG      config_server_log.py ----- Отладочная информация
2021-08-12 00:25:40,334 ----- INFO       config_server_log.py ----- Информационное сообщение
